includes/ECommerce.php

```diff
@@ -446,7 +446,7 @@ function custom_add_promotion_menu_item( $menu_items ) {
     add_submenu_page(
       'woocommerce-marketing',
       'Promotion product Page',
-      __('Promote','wp_module_ecommerce'),
+      __('Promotions','wp-module-ecommerce'),
```

Comment:
this should be in double quotes.

Reply:
@manikantakailasa Our coding standards state that we should use single quotes everywhere unless double quotes are required for string interpolation.